internal/server/openapi.go

```diff
@@ -18,20 +17,24 @@ import (
 	"github.com/infrahq/infra/api"
 )
 
-var (
-	pathIDReplacer            = regexp.MustCompile(`:\w+`)
-	funcPartialNameToTagNames = map[string]string{
-		"Grant":       "Grants",
-		"User":        "Users",
-		"Group":       "Groups",
-		"AccessKey":   "Authentication",
-		"Provider":    "Providers",
-		"Destination": "Destinations",
-		"Token":       "Destinations",
-		"Login":       "Authentication",
-		"Logout":      "Authentication",
-	}
-)
+var pathIDReplacer = regexp.MustCompile(`:\w+`)
+
+// funcPartialNameToTagNames is a sorted (alphabetically by tag name) list of
+// function name substrings to the tags associated with the operation.
+var funcPartialNameToTagNames = []struct {
+	partial string
+	tag     string
+}{
+	{partial: "AccessKey", tag: "Authentication"},
```

Comment:
Previously we were sorting the map each time. I think the reason for the sort is that we want openapi doc generation to be stable, so if an endpoint has multiple tags we want them to appear in the `operation.Tags` list in the same order. If we iterate over a map I think the order can change.

This PR changes the map to a slice, so that we don't have to sort it each time, it's always sorted in the literal.

I guess another way of making the generation stable would be to just sort the tags at the end after they are all applied to the operation.